doc/manual/42-server-commands.en.md

```diff
@@ -53,6 +53,23 @@ barman check <server_name>
 > and monitoring infrastructure. The `--nagios` option allows you
 > to easily create a plugin for Nagios/Icinga.
 
+## `generate-manifest`
+
+This command is useful when backup is created remotely and bg_basebackup is not 
+involved and `backup_manifest` file does not exist in backup.
+It will generate `backup_manifest` from backup_id if it does not already exist.
```

Comment:
I think it's worth emphasizing here that we generate the backup_manifest from the files already stored on the barman server, since that is a key difference between a barman-generated backup_manifest and a backup_manifest generated by `pg_basebackup`.

Reply:
Please check if the updated text is enough.